metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -284,6 +289,11 @@ class SnowflakeV2Config(
         description="If enabled, streams will be ingested as separate entities from tables/views.",
     )
 
+    include_procedures: bool = Field(
+        default=True,
+        description="If enabled, procedures will be ingested as separate entities from tables/views.",
```

Comment:
What if we make this more assertive and precise? Something like:

```suggestion
        description="If enabled, procedures will be ingested as pipelines/tasks.",
```